pandas/core/indexes/base.py

```diff
@@ -2320,6 +2320,11 @@ def symmetric_difference(self, other, result_name=None):
         except TypeError:
             pass
 
+        # On equal MultiIndexes the difference is empty. Therefore an empty
+        # MultiIndex is returned GH13490
+        if self.nlevels > 1 and len(the_diff) == 0:
+            return type(self)([[] for _ in range(self.nlevels)],
```

Comment:
ok, then I guess you can special case ``values=None`` in ``_shallow_copy`` to take a different path of construction (IOW don't use the MultiIndex.from_tuples). The crucial point is to propogate the meta-data.